dotnet-desktop-guide/framework/winforms/controls/multithreading-in-windows-forms-controls.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Multithreading in Controls
+description: Learn about multithreading in Windows Forms Controls, for example making your UI more responsive by performing time-consuming operations on another thread.
```

Comment:
I'd replace "for example" with "such as" here; it flows better in terms of grammar.

Reply:
Addressed.